ctgan/data_sampler.py

```diff
@@ -146,7 +146,7 @@ def dim_cond_vec(self):
 
     def generate_cond_from_condition_column_info(self, condition_info, batch):
         vec = np.zeros((batch, self._n_categories), dtype='float32')
-        id = self._discrete_column_matrix_st[condition_info["discrete_column_id"]
-                                             ] + condition_info["value_id"]
-        vec[:, id] = 1
+        id_ = self._discrete_column_matrix_st[condition_info["discrete_column_id"]
+                                              ] + condition_info["value_id"]
```

Comment:
why did this space get added?

Reply:
It was complaining that the square brackets were not in the same line, adding the space fixes it. This doesn't matter though, it will be overwritten in the base class because the multiline-container PR requires you to write that in a different way.